lib/services/pool/pool-stable.service.ts

```diff
@@ -88,7 +88,7 @@ export class PoolStableService implements PoolService {
         const userData = this.encodeJoinPool(data);
 
         if (data.zapIntoMasterchefFarm && this.pool.staking?.type === 'MASTER_CHEF' && this.pool.staking.farm) {
```

Comment:
this check is no longer adequate then, should be: `(farm && zapIntoFarm) || (gauge && zapIntoGauge)`

Reply:
Changed it!